app/src/main/java/at/roteskreuz/stopcorona/model/repositories/QuarantineRepository.kt

```diff
@@ -115,10 +115,20 @@ interface QuarantineRepository {
      */
     fun revokeLastRedContactDate()
 
+    /**
+     * Resets the last yellow contact date.
+     */
+    fun revokeLastYellowContactDate()
+
     /**
      * Get the current quarantine status.
      */
     suspend fun getQuarantineStatus(): QuarantineStatus
+
+    /**
+     * return the current Warning type based on the last Yellow and Red contact date times
```

Comment:
Capital case for the first letter and dot at the end.

Reply:
@JU5T1C3

Reply:
done